### PR TITLE
Cleaned up the spaces after comma

### PR DESCRIPTION
**Pull request by amartins**
Made some cleaning...

### CODE REVIEW

File: communities/as24115.txt [hunk 1]
```diff
@@ -3,51 +3,51 @@
...
-24115:102:xxx, Prepend 2x to Neighbor ASN
-24115:103:xx, Prepend 3x to Neighbor ASN
-24115:111:xx, Prepend 1x for Latency greater than MS
-24115:112:xx, Prepend 2x for Latency greater than MS
-24115:113:xx, Prepend 3x for Latency greater than MS
+24115:1:24115,Default Open Policy
+24115:0:24115,Default Closed Policy
+24115:0:nnn,No Export to ASN
+24115:1:nnn,Allow Export to ASN
+24115:2:nn,No Export to Neighbors above MS
```
Comment (job) [inline communities/as24115.txt:31]: are you sure you intend to switch `xx` to `nn`?

Comment (teunvink) [reply communities/as24115.txt:31]: `nn` is nothing. `xxx` is 3 digits, `nnn` is any number. `0-999` could work as well, but `xxx' seems to be the best way.

Comment (amartins) [reply communities/as24115.txt:31]: let me fix it. I'l cancel this PR.

**Comment by amartins** [comment]
Close this for now.

**Pull request closed without merging**